livedata/src/main/java/io/getstream/chat/android/livedata/request/Mapper.kt

```diff
@@ -3,13 +3,15 @@ package io.getstream.chat.android.livedata.request
 import io.getstream.chat.android.client.api.models.QueryChannelsRequest
 import io.getstream.chat.android.client.utils.FilterObject
 
-internal fun QueryChannelsPaginationRequest.toAnyChannelPaginationRequest(): AnyChannelPaginationRequest =
-    AnyChannelPaginationRequest().apply {
-        this.channelLimit = channelLimit
-        this.channelOffset = channelOffset
-        this.messageLimit = messageLimit
-        this.sort = sort
+internal fun QueryChannelsPaginationRequest.toAnyChannelPaginationRequest(): AnyChannelPaginationRequest {
+    val originalRequest = this
+    return AnyChannelPaginationRequest().apply {
+        this.channelLimit = originalRequest.channelLimit
+        this.channelOffset = originalRequest.channelOffset
+        this.messageLimit = originalRequest.messageLimit
+        this.sort = originalRequest.sort
     }
```

Comment:
We were assigning values to themselves here, referencing the wrong receiver (the extension receiver, instead of the dispatch receiver)